src/main/java/ByeCommand.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Exceute the bye command in Duke.
+ */
+
+public class ByeCommand extends Command {
+
+    /**
+     *  Bye Constructor with command string input.
```

Comment:
Could delete a space before "Bye" to ensure that the comments are aligned as seen in example.
![image](https://user-images.githubusercontent.com/39525563/64272845-c469cc00-cf72-11e9-87ad-9d5c47b263fa.png)